src/Servers/Kestrel/Core/src/Features/ISslStreamFeature.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Net.Security;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Core.Features;
+
+/// <summary>
+/// Feature to get access the connection's <see cref="SslStream" />.
+/// </summary>
+public interface ISslStreamFeature
+{
+    /// <summary>
+    /// Gets the <see cref="SslStream"/>.
```

Comment:
```suggestion
    /// Gets the <see cref="SslStream"/>.
    /// This is not available for non-TLS connections or HTTP/3.
```

Reply:
I'm not sure I follow - seems like either the comment should be on the interface or the property should be optional?

Reply:
It does belong on the interface, but I put it here because people are more likely to see it via intellasense. You could put it on both.